build/android/build.gradle

```diff
@@ -1,13 +1,18 @@
 buildscript {
-	repositories {
-		mavenCentral()
-		jcenter()
-		google()
-		maven { url 'https://maven.google.com' }
-	}
-	dependencies {
-		classpath "com.android.tools.build:gradle:3.1.3"
-	}
+    repositories {
+        maven { url 'https://maven.google.com' }
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.3'
+    }
+}
+
+allprojects {
```

Comment:
i don't see why we need this

Reply:
Without this, Android Studio could not successfully complete the build.